src/Sulu/Bundle/AdminBundle/Resources/js/components/withContainerSize/withContainerSize.js

```diff
@@ -54,7 +54,15 @@ export default function withContainerSize(Component: ReactClass<*>, containerCla
         }
     }
 
-    WithContainerSizeComponent.displayName = `withContainerSize(${Component.displayName || Component.name})`;
+    const componentName = (typeof Component.displayName === 'string'
```

Comment:
@chirimoya How should we solve that? We have to do something like that in order to be type safe. Feels like something that could go into a service, right? The same code is needed for every HOC, and already used twice.